### PR TITLE
Add support for files with class properties

### LINKED ISSUE #64: Add support for experimental `classProperties` Babel plugin
```
 $ yarn lint:i18n
yarn run v1.17.3
$ npx ember-intl-analyzer
[1/4] 🔍  Finding JS and HBS files...
[2/4] 🔍  Searching for translations keys in JS and HBS files...
SyntaxError: This experimental syntax requires enabling the parser plugin: 'classProperties' (6:21)
    at Parser.raise (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:6325:17)
    at Parser.expectPlugin (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:7647:18)
    at Parser.parseClassProperty (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:10830:12)
    at Parser.pushClassProperty (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:10795:30)
    at Parser.parseClassMemberWithIsStatic (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:10734:14)
    at Parser.parseClassMember (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:10668:10)
    at withTopicForbiddingContext (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:10623:14)
    at Parser.withTopicForbiddingContext (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:9702:14)
    at Parser.parseClassBody (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:10600:10)
    at Parser.parseClass (/home/lprestonsegoiii/.npm/_npx/12695/lib/node_modules/ember-intl-analyzer/node_modules/@babel/parser/lib/index.js:10574:22)
```

is there a way to tell this to use my project's babel config?
I'm using typescript w/ octane

### PR DESCRIPTION
**Pull request by jmonster**
closes #64

### CODE REVIEW

**Comment by Turbo87** [comment]
thanks @jmonster!

can you add a test case for this so that we don't regress on this issue? 🙏

**Comment by jmonster** [comment]
@Turbo87 I don't mean to neglect this PR but I had a difficult time coming up with a simple way to test this change. Could you offer any guidance for how you imagine this property being tested?

I attempted to simply use the new class syntax in a test but I believe it passed with and without the change from this PR 🤷‍♂

**Comment by Turbo87** [comment]
> I attempted to simply use the new class syntax in a test but I believe it passed with and without the change from this PR

that would have been my idea. seems surprising that it doesn't work. do you have any idea why it works without the change?

**Comment by jmonster** [comment]
@Turbo87 I updated this PR to have a test, but note as long as the file contains `this.intl.t('js-translation');` -- even if I precede it with `throw new Error("anything")` or `return false` -- the tests pass. Removing that line, however, yields test failures.

Adding `classProperties` (the original change in this PR) has no effect on these tests passing/failing

Any advice how to proceed?

File: fixtures/no-issues/app/controllers/application.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import Controller from '@ember/controller';
+import { computed } from '@ember/object';
 
-export default Controller.extend({
-  foo: computed('intl.locale', function() {
+export default class ApplicationController extends Controller {
+  @computed('intl.locale')
+  foo() {
```
Comment (Turbo87) [inline fixtures/no-issues/app/controllers/application.js]: I think that's a method, not a class property 🤔

Comment (Turbo87) [reply fixtures/no-issues/app/controllers/application.js]: you would have to use something like `@tracked foo;`

Comment (jmonster) [reply fixtures/no-issues/app/controllers/application.js]: nailed it! tests now fail without `classProperties` and pass with it 🙃 

PR updated

**Pull request merged**